gammapy/background/__init__.py

```diff
@@ -10,3 +10,4 @@
 from .template import *
 from .kernel import *
 from .models import *
+from .make import *
```

Comment:
I think make is too generic a name, plus it would be good to have the class and functions operating on that class (if any remain standalone) in the same file.
So how about renaming `make.py` -> `cube_model.py` and moving the `CubeBackgroundModel` class here?